.github/workflows/ci.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Upload Integration Log File
         uses: actions/upload-artifact@v1
         with:
-          name: integration-test.log
+          name: integration-test-${{ github.event.pull_request.head.sha }}.log
```

Comment:
This CI script can be triggered with `push` event as well, so this might not work for all cases. 
It's better to rely on `GITHUB_SHA` env: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables